samples/samples/snippets.py

```diff
@@ -2510,6 +2510,35 @@ def update_venues(transaction):
     # [END spanner_set_transaction_tag]
 
 
+def set_transaction_timeout(instance_id, database_id):
+    """Executes a transaction with a transaction timeout."""
+    # [START spanner_transaction_timeout]
+    # instance_id = "your-spanner-instance"
+    # database_id = "your-spanner-db-id"
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+    database = instance.database(database_id)
+
+    def read_then_write(transaction):
+        # Read records.
+        results = transaction.execute_sql(
+            "SELECT SingerId, FirstName, LastName FROM Singers ORDER BY LastName, FirstName"
+        )
+        for result in results:
+            print("SingerId: {}, FirstName: {}, LastName: {}".format(*result))
+
+        # Insert a record.
+        row_ct = transaction.execute_update(
+            "INSERT INTO Singers (SingerId, FirstName, LastName) "
+            " VALUES (100, 'George', 'Washington')"
+        )
+        print("{} record(s) inserted.".format(row_ct))
+
+    database.run_in_transaction(read_then_write, timeout_secs=60)
```

Comment:
add a comment here , that setting timeout of 60 secs

Reply:
done